### ISSUE TITLE
Convert student explorer to use JSON instead of env for configuration

### ISSUE THREAD

**Issue by jonespm** [tl-its-umich-edu/student_explorer#119]
We recently switched MyLA over from environment to JSON for a number of reasons, see them over here:  https://github.com/tl-its-umich-edu/my-learning-analytics/issues/546

We should do the same for this project to ease on deployment and improve security.

**Comment by jonespm** [comment]
It might be better to just skip this and go for a live-settings
https://djangopackages.org/grids/g/live-setting/

The most popular/updated there are django-constance and django-dynamic-preferences

**Comment by anantakash73** [comment]
Hi, I'm looking into django-constance to migrate to live-settings. 

Seems like that is a lot less work than using JSON instead of env. Any pointers would be appreciated.

**Comment by jonespm** [comment]
Yeah, I agree. I haven't looked at it yet and don't know your level of Django experience, but even if *some* of the settings were migrated out of the settings.py into a live settings like this I'd consider it a success. 

I like how Constance looks https://django-constance.readthedocs.io/en/latest/

And I'd probably just go with the database backend to be easier for now (though we have been looking into using Redis more in production).

It looks pretty straightforward.

"Modify your settings.py. Add 'constance' to your INSTALLED_APPS, and move each key you want to turn dynamic into the CONSTANCE_CONFIG section, like this:"

So try to install it, move a few settings we have defined there now (like maybe the one with the EMAIL_PREFIX to start, and see how it goes? 

The only ones I wouldn't move out would be the database configuration properties (as they're required to actually get this package started) or anything else security related. But my goal is to remove the unsupported decouple library. 

Thanks!

**Comment by pbradtke** [comment]
Hi, we're two students in EECS: 481 (Software Engineering) and we're interested in tackling this issue as part of our final project. We both have significant experience developing webapps using Django and would love to help out by integrating Constance. Is this still an issue which would be beneficial to the project?

**Comment by jonespm** [comment]
Sure! It looks like this was one another student in the past was interested in but didn't get anywhere. I think even if this got started with a few settings it would be appreciated. Thanks!
